spacetemplate/assets/legacy.yaml

```diff
@@ -87,6 +87,42 @@ work_item_types:
   - *experienceID
   - *valuePropositionID
 
+board_config:
+
+- id: "24181b5c-713f-4bef-a19f-45240875da92"
+  name: Scenarios Board
+  description: This is the default board config for the legacy template (Scenarios).
+  context: "679a563c-ac9b-4478-9f3e-4187f708dd30"
+  contextType: "TypeLevelContext"
+  columns:
+  - id: "b4edad70-1d77-4e5a-b973-0f0d599fd20d"
+    title: "New"
+    order: 0
+    transRuleKey: "updateStateFromColumnMove"
+    transRuleArguments: "{ metaState: 'mNew' }"
+  - id: "b4edad70-1d77-4e5a-b973-0f0d599fd20d"
```

Comment:
Please change the `id` of `In Progress`. It is same as `id` of `New`.

Reply:
Yes, fixed that already locally.